src/colvarbias_restraint.h

```diff
@@ -258,7 +258,6 @@ class colvarbias_restraint_harmonic_walls
   colvarbias_restraint_harmonic_walls(char const *key);
   virtual int init(std::string const &conf);
   virtual int update();
-  virtual void communicate_forces();
```

Comment:
Because I've removed the corresponding implementation for this class. The compiler complained.

Reply:
Ah yes, I forgot that you moved the code to the base class. (Working from home today due to sick child). It may not be a bad idea to remove `virtual` altogether for this function, which was made so to deal precisely with this issue in cb80f13679e90feb23b3ea342e38733c5f430c4f.

Reply:
Good point! Will do that.